test/test_scenario_3.py

```diff
@@ -0,0 +1,27 @@
+import os
```

Comment:
I assume ci testing of the python options is not needed ad we call it ourselfbes in a pythonic way underneat.